### ISSUE TITLE
Make focus indication more visible: 3

### ISSUE THREAD

**Issue by paregorios** [isawnyu/isaw.web#249]
> Anyone who relies on the keyboard to operate a page needs to visually determine where the focus is set at any point in time. **When a user interface control receives focus, a visible border should be displayed around it, or the color should noticeably change.** The width of the border should increase to 3px in width, and the color should be chosen to pass color contrast guideline.

Currently, our navigation provides a modest brightness change to indicate focus, but it is insufficiently noticeable.

![screen shot 2018-04-24 at 2 35 53 pm](https://user-images.githubusercontent.com/263285/39209910-d21c18d2-47cc-11e8-92c5-446525a6aac6.png)

Traceability: [Microassist Report 1 > WCAG Compliance > Focus Indication](https://docs.google.com/document/d/10cfUFx-o_kSGBFKCe4MtL45cyc-c-Gwaz3Dxlrp8Qgg/edit#bookmark=id.oqxdqqok7z3z)

[WCAG 2.4.7 Focus Visible: Any keyboard operable user interface has a mode of operation where the keyboard focus indicator is visible. (Level AA)](https://www.w3.org/WAI/WCAG20/quickref/#qr-navigation-mechanisms-focus-visible)

**Comment by paregorios** [comment]
Make the following CSS changes for navigation:

```css
#portal-top #main-navigation ul#primary li a {
    ....
    line-height: 72px; /* changed from 78px */
    ....
}
#portal-top #main-navigation ul#primary li a:hover, 
#portal-top #main-navigation ul#primary li a:focus {
    ...
    border: 3px solid blue !important;  /* new directive */
}
#portal-top #main-navigation ul#primary li#portaltab-graduate-studies a {
    ...
    border: 3px solid #6d6054; /* new directive */
}
#portal-top #main-navigation ul#primary li#portaltab-visiting-scholars a {
    ....
    border: 3px solid #9d3a20; /* new directive */
}
#portal-top #main-navigation ul#primary li#portaltab-events a {
    ....
    border: 3px solid #66232b; /* new directive */
}
#portal-top #main-navigation ul#primary li#portaltab-exhibitions a {
    ...
    border: 3px solid #9e6a4a; /* new directive */
}
#portal-top #main-navigation ul#secondary li a {
    ...
    line-height: 43px; /* changed from 49 */
    border: 3px solid transparent; /* new directive */
    ...
}
#portal-top #main-navigation ul#secondary li a:hover, 
#portal-top #main-navigation ul#secondary li a:focus {
    ...
    border: 3px solid blue !important; /* new directive */
}
```

## Here is the effect we should see for main nav (focus/hover is on "exhibitions"):

![screen shot 2018-12-03 at 11 25 09 am](https://user-images.githubusercontent.com/263285/49390564-f05bd680-f6ee-11e8-9c2f-184ffb6800b4.png)

## Here is the effect we should see for secondary nav (focus/hover is on "about isaw"):

![screen shot 2018-12-03 at 11 32 14 am](https://user-images.githubusercontent.com/263285/49390645-1da88480-f6ef-11e8-8e97-621c1ee56c7a.png)

**Comment by alecpm** [comment]
This is live on staging (whenever it's back online)

**Comment by paregorios** [comment]
This looks good. Please deploy to production.
